framework/doc/content/source/meshmodifiers/SidesetAroundSubdomainUpdater.md

```diff
@@ -2,10 +2,13 @@
 
 ## Description
 
-The `SidesetAroundSubdomainUpdater` mesh modifier updates a sideset during a simulation by reassigning it based on a pair of subdomain sets, the [!param](/MeshModifiers/SidesetAroundSubdomainUpdater/inner_subdomains) and [!param](/MeshModifiers/SidesetAroundSubdomainUpdater/outer_subdomains). The sideset will be updated to comprise all sides along `inner_subdomains` that are neighboring `outer_subdomains` or that have no neighbor. This dynamic update can be useful in conjunction with subdomain update user objects such as [CoupledVarThresholdElementSubdomainModifier](CoupledVarThresholdElementSubdomainModifier.md).
+The `SidesetAroundSubdomainUpdater` mesh modifier updates a sideset during a simulation by reassigning it based on a pair of subdomain sets, the [!param](/MeshModifiers/SidesetAroundSubdomainUpdater/inner_subdomains) and [!param](/MeshModifiers/SidesetAroundSubdomainUpdater/outer_subdomains). The sideset will be updated to comprise all sides along `inner_subdomains` that are neighboring `outer_subdomains` or that have no neighbor. This dynamic update can be useful in conjunction with subdomain update user objects such as [CoupledVarThresholdElementSubdomainModifier](CoupledVarThresholdElementSubdomainModifier.md).  The element sides on which this object acts can be limited using the [!param](/MeshModifiers/SidesetAroundSubdomainUpdater/mask_side) parameter, as shown in:
```

Comment:
let s explain this mask more. 

The mask is restricting the sides which can be added to the moving side as the boundary moves.
OR
The mask is restricting the sides which can be removed from the moving side as the boundary moves.

Reply:
I think it s the former